### PR TITLE
Implement saving camera reads to device

### LINKED ISSUE #111: Log numbers to the device

### PR DESCRIPTION
**Pull request by kantoniak**
Closes #111.

### CODE REVIEW

File: camera/rpi-files/data_archiver.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                self._path, ex1,
+            )
+            raise RuntimeError(msg)
+
+        field_names = [
+            'timestamp',
+            EventType.monitoring_started.name,
+            EventType.monitoring_ended.name,
+            EventType.people_entered.name,
+            EventType.people_left.name,
```
Comment (Rhantolq) [inline camera/rpi-files/data_archiver.py:99]: Should this be "people" instead of "person"?

Comment (kantoniak) [reply camera/rpi-files/data_archiver.py]: It should. Fixed.

File: camera/rpi-files/data_archiver.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+            RuntimeError: if path already exists or could not create file
+        """
+        if not force_overwrite and os.path.exists(self._path):
+            msg = 'Path `{0}` already exists'.format(self._path)
+            raise RuntimeError(msg)
+
+        try:
+            self._archive_file = open(self._path, 'w+')  # noqa: WPS515
+        except OSError as ex1:
+            msg = 'Could not open path `{0}` for writing: {1}'.format(
```
Comment (avan1235) [inline camera/rpi-files/data_archiver.py:89]: think of using f strings if using proper version of python

File: camera/rpi-files/data_archiver.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+            EventType.person_entered.name,
+            EventType.person_left.name,
+        ]
+        try:
+            self._writer = csv.DictWriter(
+                self._archive_file, fieldnames=field_names,
+            )
+        except csv.Error as ex2:
+            msg = 'Error creating csv.writer: {0}'.format(ex2)
+            raise RuntimeError(msg)
```
Comment (avan1235) [inline camera/rpi-files/data_archiver.py:107]: How to you plan to handle these Runtime errors? Is it gonna try to restart / reboot or just hang after sending some signal

Comment (kantoniak) [reply camera/rpi-files/data_archiver.py:107]: I think we can close the device. All the exceptions are handled, but if camera can't do it's work, I wouldn't try to restart.

**Pull request merged**